YJE/Day16/app/app.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+// 모듈
+const dotenv = require("dotenv");
```

Comment:
현재 require()를 이용해 모듈을 불러오고 계신 점, 잘 작성해주셨습니다!
절대 잘못된 방식은 아니지만, ES6의 import 문법도 함께 익혀두시면
앞으로 더 깔끔하고 일관성 있는 코드를 작성하는 데 도움이 됩니다.

예를 들어 아래와 같이 바꿔볼 수 있습니다:

```suggestion
import TodoModel from 'dotenv';
```

참고해보면 좋을거 몇개 걸어놓겠습니다.
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Statements/import

https://inpa.tistory.com/entry/NODE-%F0%9F%93%9A-require-%E2%9A%94%EF%B8%8F-import-CommonJs%EC%99%80-ES6-%EC%B0%A8%EC%9D%B4-1

Reply:
조언 감사합니다!
현재는 require()를 주로 사용하고 있는데, 앞으로 가독성과 일관성을 위해 ES6 문법으로 전환하는 연습도 해보겠습니다.